cmd/jwt.go

```diff
@@ -22,8 +22,8 @@ import (
 	"net/http"
 	"time"
 
-	jwtgo "github.com/dgrijalva/jwt-go"
-	jwtreq "github.com/dgrijalva/jwt-go/request"
+	jwtgo "github.com/golang-jwt/jwt"
+	jwtreq "github.com/golang-jwt/jwt/request"
```

Comment:
Before we do this, I think the `golang-jwt/jwt` should fix the incompatible issue with go.mod

Reply:
They have added go modules support but had to revert it. See https://github.com/golang-jwt/jwt/pull/26 and https://github.com/golang-jwt/jwt/issues/17 for details. It doesn't have any external dependencies itself therefore there are no downsides except the visually unpleasing `+incompatible` suffix